src/button-group/types.ts

```diff
@@ -84,4 +84,6 @@ export type StateReducerT = (
 ) => StateT;
 
 // general
-type ClickHandlerT = (event: SyntheticEvent<HTMLButtonElement>, index: number) => unknown;
+type ClickHandlerT = (...args: any[]) => void;
+// todo: causes type errors, and does not look right
+// type ClickHandlerT = (event: SyntheticEvent<HTMLButtonElement>, index: number) => unknown;
```

Comment:
should be correct based on usage here https://github.com/uber/baseweb/pull/5009/files#diff-3a7f2280f8ed9c3b284d87b0a18297b198b4f35f4944334fbf2de980b78e7a8dR118 What is the error?